sci-libs/ipopt/ipopt-3.12.11.ebuild

```diff
@@ -0,0 +1,90 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+FORTRAN_NEEDED="mumps"
+
+inherit autotools fortran-2 toolchain-funcs
+
+MY_PN=${PN^}
+MY_P=${MY_PN}-${PV}
+
+DESCRIPTION="Interior-Point Optimizer for large-scale nonlinear optimization"
+HOMEPAGE="https://projects.coin-or.org/Ipopt/"
+SRC_URI="http://www.coin-or.org/download/source/${MY_PN}/${MY_P}.tgz"
+
+LICENSE="EPL-1.0 hsl? ( HSL )"
+SLOT="0/1"
+KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux"
+IUSE="doc examples hsl lapack mpi mumps static-libs test"
+
+RDEPEND="
+	virtual/blas
+	hsl? ( sci-libs/coinhsl:0= )
+	lapack? ( virtual/lapack )
+	mumps? ( sci-libs/mumps:0=[mpi=] )"
+DEPEND="${RDEPEND}
+	virtual/pkgconfig
+	doc? ( app-doc/doxygen[dot] )
+	test? ( sci-libs/coinor-sample sci-libs/mumps )"
+
+S="${WORKDIR}/${MY_PN}-${PV}/${MY_PN}"
+
+src_prepare() {
+	if use mpi ; then
+		export CXX=mpicxx FC=mpif77 F77=mpif77 CC=mpicc
+	fi
+	default
+}
+
+src_configure() {
+	# needed for the --with-coin-instdir
+	dodir /usr
+	local myeconfargs
```

Comment:
What is the reason for moving the standard args out of the nice array and regressing them with `\` chars towards `econf`? I'd much rather see a clean `econf "${myeconfargs[@]}"` at the end. Also, having the related cmake arg right there is much nicer wrt the comment above.